packages/@lwc/engine-core/src/framework/restrictions.ts

```diff
@@ -316,36 +313,6 @@ function getLightningElementPrototypeRestrictionsDescriptors(
             },
         }),
     };
-
-    forEach.call(getOwnPropertyNames(globalHTMLProperties), (propName: string) => {
-        if (propName in proto) {
-            return; // no need to redefine something that we are already exposing
-        }
-        descriptors[propName] = generateAccessorDescriptor({
-            get(this: LightningElement) {
-                const { error, attribute } = globalHTMLProperties[propName];
-                const msg: string[] = [];
-                msg.push(`Accessing the global HTML property "${propName}" is disabled.`);
-                if (error) {
-                    msg.push(error);
-                } else if (attribute) {
-                    msg.push(`Instead access it via \`this.getAttribute("${attribute}")\`.`);
-                }
-                logError(msg.join('\n'), getAssociatedVM(this));
-            },
-            set(this: LightningElement) {
-                const { readOnly } = globalHTMLProperties[propName];
-                if (readOnly) {
-                    logError(
-                        `The global HTML property \`${propName}\` is read-only.`,
-                        getAssociatedVM(this)
-                    );
-                }
-            },
-        });
-    });
-
-    return descriptors;
```

Comment:
This is the bulk of the change. This is where I tried a lot of zany solutions to extend the `HTMLElement` prototype while adding the logging, but none of them worked. So let's just get rid of this logging.